camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/mapred/EtlMultiOutputRecordWriter.java

```diff
@@ -30,6 +30,7 @@
   private String currentTopic = "";
   private long beginTimeStamp = 0;
   private static Logger log = Logger.getLogger(EtlMultiOutputRecordWriter.class);
+  private static Counter topicSkipOldCounter = null;
```

Comment:
I think this can be defined as 'private final' instance variable, so it's guaranteed to be there, there is no need to do a lazy-initialization.